### ISSUE TITLE
move login func to server

### ISSUE THREAD

**Issue by duncangraham** [freindst/Argo-Project-V0#28]
I've been trying to move the login functionality to the server, so I can set a currentUser session variable. 

I'm getting the error: 'Error: It is not possible to log in on a server environment.' which seems insane to me. Why would Parse only allow you to login on the client? I'll keep trying to debug, but do you have any ideas @freindst?

here's the current work: https://github.com/freindst/Argo-Project-V0/tree/login

**Comment by freindst** [comment]
I am outside for sports. I will be back in the evening.

**Comment by freindst** [comment]
I have tried all sorts of attempts, but I keep on meeting the same error message as you. Personally, I agree with parse not post username and password to server, as POST is not safe either.
After I gave up the idea of running user authentication on server side, I add a piece of Javascript code to login page, and successfully login will post userID back to node. The rest of the work is simple.
app.post('/login', function (req, res) {
	req.session.currUserID = req.body.userID;
	res.redirect("/");
})
